app/scripts/metamask-controller.js

```diff
@@ -6194,7 +6219,7 @@ export default class MetamaskController extends EventEmitter {
    */
   _onStateUpdate(newState) {
     this.isClientOpenAndUnlocked = newState.isUnlocked && this._isClientOpen;
-    this._notifyChainChange();
+    // this._notifyChainChange();
```

Comment:
should this still be commented out?

Reply:
no it was extra noisy sending notifications across the wire via externally_connectable, not sure if this is what we want for the new pipeline

Reply:
i like my pipes noisy. how else do you know if they're still working? /s